pgml-dashboard/src/components/navbar_web_app/template.html

```diff
@@ -48,7 +48,7 @@
             </li>
   
             <li class="nav-item d-flex align-items-center">
-              <a class="nav-link p-lg-0" href="/docs/guides/setup/quick_start_with_docker">Docs</a>
+              <a class="nav-link p-lg-0" href="/docs/guides/README">Docs</a>
```

Comment:
Not very SEO friendly imo

Reply:
Yeah, but we can't control this lever in gitbook... and then again... it is kinda devvy.